src/test/tcp/test_tcp.c

```diff
@@ -49,6 +61,37 @@ static void _mylog(const char* fileName, const int lineNum, const char* funcName
     fflush(stdout);
 }
 
+// create a System V queue with key based on a file queue name
+static int get_queue(const char* queuename) {
+    key_t key = ftok(queuename, 0);
+    assert_true_errno(key != -1);
+
+    int fd = msgget(key, IPC_CREAT | 0666);
```

Comment:
No need to make this world accessible, especially since even a read from an outside process would confuse the test. 0066 is probably what we want here.

Reply:
0666 didn't work. But 0600 works. Is it ok?